github-actions/start-release/tests/data/expected_release_notes.md

```diff
@@ -0,0 +1,8 @@
+**<APP_NAME> Release Notes - Published by <PUBLISHER> <PUBLISH_DATE>**
+
+
+**Version <VERSION> - Released <INSERT_RELEASE_DATE>**
```

Comment:
Is there a separate part of the release process, outside of this PR that will look for this string and replace it with the release date?

Reply:
Since our release process is not fully automated yet, there could be some delay between when release notes are generated and when a new tarball is uploaded

My idea was to have the person uploading the tarball to Splunk Base replace the release date here, although I think its reasonable to set the release date to be the same as `<PUBLISH_DATE>` initially, since it may not need to be updated if the app is released on the same day that the release notes are created

Reply:
Makes sense